packages/components/src/popover/stories/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { boolean, select, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Popover from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
+export default { title: 'Popover', component: Popover };
+
+export const _default = () => {
+	const show = boolean( 'Example: Show', true );
+	const children = text( 'children', 'Popover Content' );
+	const animate = boolean( 'animate', false );
+	const focusOnMount = select(
+		'focusOnMount',
+		{
+			firstElement: 'firstElement',
+			container: 'container',
+		},
+		'firstElement'
+	);
+	const noArrow = boolean( 'noArrow', false );
+
+	const props = {
+		animate,
+		children,
+		focusOnMount,
+		noArrow,
+	};
+
+	if ( ! show ) {
+		return null;
+	}
+
+	return <Popover { ...props } />;
+};
+
+const DragExample = ( props ) => {
+	const [ position, setPosition ] = useState( { x: 0, y: 0 } );
+	const [ isDragging, setDragging ] = useState( false );
+	const { label, content, ...restProps } = props;
+
+	const updatePosition = ( event ) => {
+		if ( ! isDragging ) {
+			return false;
+		}
+		const { movementX, movementY } = event;
+		setPosition( {
+			x: position.x + movementX,
+			y: position.y + movementY,
+		} );
+	};
+
+	const startDragging = () => setDragging( true );
+	const stopDragging = () => setDragging( false );
```

Comment:
We are generating new functions startDragging, stopDragging, and updatePosition. Could we wrap these functions with useCallback to avoid that?

Reply:
@jorgefilipecosta Ah yes! Thank you. Will make that update